gammapy/image/mask.py

```diff
@@ -19,6 +17,7 @@
     'make_tevcat_exclusion_mask'
 ]
 
+MAX_VALUE = 1e10
```

Comment:
I'd suggest to move this `MAX_VALUE` inside the `distance_image` method.
This is closer to where it's used and thus code that's easier to read / maintain.
If we ever need it somewhere else (probably never) we can then make it global.